pmd-javascript/src/test/java/net/sourceforge/pmd/lang/ecmascript/rule/codestyle/UnnecessaryBlockTest.java

```diff
@@ -1,11 +1,32 @@
-/**
- * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
- */
-
 package net.sourceforge.pmd.lang.ecmascript.rule.codestyle;
 
 import net.sourceforge.pmd.testframework.PmdRuleTst;
 
-class UnnecessaryBlockTest extends PmdRuleTst {
-    // no additional unit tests
+public class UnnecessaryBlockTest extends PmdRuleTst {
+    @Override
+    public void setUp() {
+        // Set up any required configurations or resources before running the tests
+    }
+
+
+    public void testUnnecessaryBlockInImportStatement() {
+        String code = "import { foo } from 'bar';\n" +
+                      "{\n" +
+                      "    // Unnecessary block\n" +
+                      "}\n";
+
+        // Assert that the PMD rule does not flag the unnecessary block within import statement
+        addSourceCodeTest(code, 0);
+    }
+
+    public void testUnnecessaryBlockInDestructuringAssignment() {
+        String code = "const { a, b } = obj;\n" +
+                      "{\n" +
+                      "    // Unnecessary block\n" +
+                      "}\n";
+
+        // Assert that the PMD rule does not flag the unnecessary block within destructuring assignment
+        addSourceCodeTest(code, 0);
+    }
```

Comment:
thanks for the PR!

Please, move these test-scenarios to the [`UnnecessaryBlock.xml`](https://github.com/pmd/pmd/blob/master/pmd-javascript/src/test/resources/net/sourceforge/pmd/lang/ecmascript/rule/codestyle/xml/UnnecessaryBlock.xml) file.

You can look at existing samples for reference, or check the [Test XML Reference](https://docs.pmd-code.org/latest/pmd_userdocs_extending_testing.html#test-xml-reference)